### PR TITLE
Minor refactoring: data preprocessing script

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

File: insights_engine/training/data_preprocessing.py [hunk 1]
```diff
@@ -106,14 +106,47 @@ def parse_godeps(mercator_op):  # pragma: no cover
...
+    for idx, manifest in enumerate(manifests_all):
+        manifest = set(manifest)
+        duplicate = False
+        for man_c in manifest_list_new:
+            if manifest == man_c:
+                duplicate = True
+                break
+        if not duplicate:
+            manifest_list_new.append(manifest)
+    print(len(manifest_list_new))
```
Comment (msrb) [inline insights_engine/training/data_preprocessing.py:123]: Leftover from debugging?

Comment (tisnik) [reply insights_engine/training/data_preprocessing.py:123]: might be, I did not want to modify the logic, just refactor it :)

**Comment by tisnik** [comment]
cool, the CC is down to manageable numbers ;)

File: insights_engine/training/data_preprocessing.py (hunk 1 above)
Comment (rootAvish) [reply insights_engine/training/data_preprocessing.py:123]: Looks to be the case @msrb I've filed an issue: https://github.com/fabric8-analytics/f8a-golang-insights/issues/16 and I'll change all of these to logging statements.

**Review by rootAvish — APPROVED** [review]
I see why this was a problem, fair enough.

**Pull request merged**